tests/performance/nearest_neighbor_distance_metrics_and_types/nearest_neighbor_distance_metrics.rb

```diff
@@ -13,16 +13,21 @@ def test_metrics_and_types
     set_description('Benchmark distance metrics and types using 500K documents. Exact nearest neighbor search.')
     deploy_app(SearchApp.new.sd(selfdir + 'vector.sd'))
     start
-
+  
     feed_file = 'vectors.perf.json.zstd'
     remote_file = "https://data.vespa.oath.cloud/tests/performance/#{feed_file}"
     local_file =  dirs.tmpdir + feed_file
+    local_feed_file =  dirs.tmpdir + "feed.json"
     cmd = "wget -O'#{local_file}' '#{remote_file}'"
     puts "Running command #{cmd}"
     result = `#{cmd}`
     puts "Result: #{result}"
-    run_feeder(local_file, [])
 
+    cmd2 = "zstdcat '#{local_file}' > '#{local_feed_file}'"
+    puts "Running command #{cmd2}"
+    result2 = `#{cmd2}`
+    puts "Result2: #{result2}"
+    run_feeder(local_feed_file, [])
```

Comment:
You can use feed_stream or run_stream_feeder. See
https://github.com/vespa-engine/system-test/blob/master/tests/performance/tensor_update/tensor_update.rb

Something like
feed_stream("curl 'url' | zstdcat")